SignalMessaging/environment/migrations/OWS105AttachmentFilePaths.m

```diff
@@ -32,7 +32,7 @@ - (void)runUpWithTransaction:(YapDatabaseReadWriteTransaction *)transaction
                                               [attachmentStreams addObject:attachmentStream];
                                           }];
 
-    DDLogInfo(@"Saving %zd attachment streams.", attachmentStreams.count);
+    DDLogInfo(@"Saving %lu attachment streams.", (unsigned long)attachmentStreams.count);
```

Comment:
Can't you use `%zu` and skip the cast?

Reply:
Nope, that results in a warning on 32bit. 

I previously explored avoiding the cast with a %tu which sort of "happens" to work on 32/64, but opted not to continue that practice: https://github.com/signalapp/Signal-iOS/pull/3385#discussion_r186450824